indico/MaKaC/conference.py

```diff
@@ -3798,11 +3728,14 @@ def clone( self, startDate, options, eventManager=None, userPerformingClone = No
             conf.setAccessKey(self.getAccessKey())
             conf.setModifKey(self.getModifKey())
         # Access Control cloning
-        if options.get("access",False) :
+        if options.get("access", False):
             conf.setProtection(self.getAccessController()._getAccessProtection())
-            for mgr in self.getManagerList() :
-                conf.grantModification(mgr)
-            for user in self.getAllowedToAccessList() :
+            from indico.modules.events.models.principals import EventPrincipal
+            cols = {column.key for column in inspect(EventPrincipal).column_attrs} - {'event_id', 'id'}
```

Comment:
I could probably add a util function such as `get_columns(EventPrincipal, exclude={'id', 'event_id'})` since we'll probably do this in more places when cloning things.